js/cellGenerator.mjs

```diff
@@ -223,6 +224,28 @@ function initSingleCell(cell, extraColumnName, format = CELL_FORMAT_CHECKLIST){
     }
     rowhtml.setAttribute("clid",usableId);
 
+    //img before name
+    if(format & CELL_FORMAT_SHOW_ICON){
+        let htmlIcon;
+        if(!COPYING){
+            htmlIcon = document.createElement("img");
+            htmlIcon.classList.add("itemIcon");
+            htmlIcon.loading = "lazy";
+            htmlIcon.draggable = "false";
```

Comment:
Checklist icons appear to be draggable for some reason even though they are set to not be.